Tests/MongoSwiftTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -156,17 +169,17 @@ struct UnifiedTestRunner {
                         let db = self.internalClient.anyClient.db(collData.databaseName)
                         let collOpts = MongoCollectionOptions(writeConcern: .majority)
                         let coll = db.collection(collData.collectionName, options: collOpts)
-                        try coll.drop()
+                        _ = try await coll.drop()
 
                         guard !collData.documents.isEmpty else {
-                            _ = try db.createCollection(
+                            _ = try await db.createCollection(
                                 collData.collectionName,
                                 options: CreateCollectionOptions(writeConcern: .majority)
                             )
                             continue
                         }
 
-                        try coll.insertMany(collData.documents)
+                        _ = try await coll.insertMany(collData.documents)
```

Comment:
this method is marked `@discardableResult`, so I don't think you would get a compiler warning if you remove the `_ =` part and ignore the `InsertManyResult`